tools/packaging/kernel/configs/fragments/x86_64/acpi.conf

```diff
@@ -4,6 +4,8 @@ CONFIG_X86_INTEL_PSTATE=y
 # Firecracker needs this to support `vcpu_count`
 CONFIG_X86_MPPARSE=y
 
+CONFIG_X86_64_ACPI_NUMA=y
```

Comment:
Should that be the default for Kata? What is the cost in terms of kernel size / complexity?

Reply:
That is a fair point, let's make this configurable like ENABLE_NUMA and then only build for targets that explicitly require it